app/models/customer.py

```diff
@@ -0,0 +1,32 @@
+from flask import current_app
+from sqlalchemy.orm import backref
+from app import db
+from sqlalchemy import DateTime
+from datetime import datetime
+from sqlalchemy.orm import relation, relationship
+
+class Customer(db.Model):
+    __tablename__ = 'customers'
+    customer_id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    phone = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, default=datetime.utcnow())
+    # videos_checked_out_count = db.Column(db.Integer, default=0)
+    # creates a list of checked out videos
+    videos = db.relationship('Video', back_populates='customers', secondary='rentals')
+    # rentals = db.relationship('Rental', backref='rental', lazy=True)
```

Comment:
I would swap these out. Have Rental be your throughway to get to Video. We can get rid of the relationship between customer and video; again, that should be done through our rental table.

```suggestion
    rentals = db.relationship('Rental', backref='rental', lazy=True)
```